salon_creator/lib/screens/lesson_list_screen.dart

```diff
@@ -0,0 +1,140 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:salon_creator/common/color.dart';
+import 'package:salon_creator/firebase/database.dart';
+
+class MemberListScreen extends StatefulWidget {
+  const MemberListScreen({Key key}) : super(key: key);
+
+  @override
+  _MemberListScreenState createState() => _MemberListScreenState();
+}
+
+class _MemberListScreenState extends State<MemberListScreen> {
+  @override
+  Widget build(BuildContext context) {
+    final screenWidth = MediaQuery.of(context).size.width;
+    return StreamBuilder<QuerySnapshot<Map<String, dynamic>>>(
+      stream: FirebaseFirestore.instance
+          .collection(DbHandler.usersCollection)
+          .where("salons",
+              arrayContains: FirebaseAuth.instance.currentUser.email)
+          .snapshots(),
+      builder: (BuildContext context, snapshot) {
+        if (!snapshot.hasData) {
+          return LinearProgressIndicator();
+        }
+        if (snapshot.hasData) {
+          return Expanded(
+            child: Container(
+              padding: EdgeInsets.symmetric(horizontal: screenWidth * 0.02),
+              child: Column(
+                children: [
+                  _buildNumberOfMemberText(snapshot),
+                  Divider(
+                    color: Color.fromRGBO(218, 218, 218, 1),
+                    thickness: 1,
+                    height: screenWidth * 0.05,
+                  ),
+                  memberAvatar(snapshot),
+                ],
+              ),
+            ),
+          );
+        }
+        return Container();
+      },
+    );
+  }
+
+  Widget memberAvatar(
+      AsyncSnapshot<QuerySnapshot<Map<String, dynamic>>> snapshot) {
+    List<Widget> userList = [];
+    snapshot.data.docs.map((e) {
+      userList.add(
+        GestureDetector(
+          onTap: () {},
+          child: Container(
+            child: Column(
+              children: [
+                _buildMemberIcon(e),
+                SizedBox(height: 6),
+                Text(
+                  e.data()['profile']['name'],
+                  style: TextStyle(fontSize: 12),
```

Comment:
Please use themes to do this. Define a new theme element (heading5, for example) and use that to style this element. Please check code in the adjacent files for reference.